src/Pages/Shared/LinkedCIDetailsModal/LinkedCIDetailsModal.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import {
+    Drawer,
+    InfoColourBar,
+    Pagination,
+    SearchBar,
+    SortableTableHeaderCell,
+    SortingOrder,
+    getCommonSelectStyle,
+} from '@devtron-labs/devtron-fe-common-lib'
+import ReactSelect from 'react-select'
+import { LinkedCIDetailsModalProps } from './types'
+import { ReactComponent as Info } from '../../../assets/icons/ic-info-filled.svg'
+import { ReactComponent as Close } from '../../../assets/icons/ic-close.svg'
+
+import AppListData from './constants'
+
+const commonStyles = getCommonSelectStyle()
+
+export interface AppListRowDataType {
+    appName: string
+    environment: string
+    triggerMode: string
+    deploymentStatus: string
+}
+
+const AppListRow = ({ appName, environment, triggerMode, deploymentStatus }: AppListRowDataType) => {
+    return (
+        <div className="flexbox pl-20 pr-20 pt-8 pb-8 dc__align-self-stretch dc__align-center dc__gap-16 fs-13 fw-4 dc__content-space">
```

Comment:
same styling issue

Reply:
color is missing

Reply:
hyperlinking is missing. We should potentially use Link component from react router dom